go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/ipfs/go-path
 
+go 1.14
+
```

Comment:
Any complaints with this bump? Supporting ancient go versions seems unnecessary.